README.md

```diff
@@ -1,70 +1,187 @@
+## 💻 Getting Started <a name="getting-started"></a>
 
-### Analyzing the Bundle Size
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+In order to run this project you need:
 
-### Making a Progressive Web App
+- A web browser to view output e.g [Google Chrome](https://www.google.com/chrome/).
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- Install the `npm` package manager use this [to install both node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
 
-### Advanced Configuration
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Clone this repository to your desired folder or download the Zip folder:
 
-### Deployment
+```
+git clone https://github.com/sheezu900/Math-project.git
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+- Navigate to the location of the folder in your machine:
 
-### `npm run build` fails to minify
+**``you@your-Pc-name:~$ cd desktop``**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Install
+
+To install all dependencies, run:
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+- After Cloning this repo to your local machine.
+- To get it running on your default browser and local host, run:
+
+1. Open the index with live server from the code editor
+2. Or you can double click on the index to open it in the browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- Track HTML linter errors run:
+```
+npx hint .
+```
+- Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Future Feature <a name="future feature"></a>
+
+-   In future we will improve design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shah Zaman**
+
+-   GitHub: [@githubhandle](https://github.com/sheezu900/)
+-   LinkedIn: [@Linkedin](https://www.linkedin.com/in/shahzaman900/)
+
+<p align="right">(<a href="#live-demo">back to top</a>)</p>
+
+
+## 🎥 Project Presentation
+
+-   Project Presentation: will add soon
```

Comment:
- [ ] [OPTIONAL] Hello @sheezu900 , let's remove this section from the README since the project presentation has not been recorded and the project hasn’t been deployed as well. This will help keep our README file professional as possible.